src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -121,6 +121,7 @@ public function configureViews(ViewCollection $viewCollection): void
             new ToolbarAction(
                 'sulu_admin.type',
                 [
+                    'sort_by_title' => true,
```

Comment:
Don't you think it would make more sense to name this option `sort_by`? I think it would scale better, because we could use `key` or `title` as a property, and it would still work in the future if we might have other properties to sort by. Could e.g. imagine that there will be a `order` property in the future, so that people can decide completely on their own how they want to order it.

Reply:
sounds good - adjusted this 🙂